lib/core/stream.nit

```diff
@@ -798,11 +836,10 @@ class BytesReader
 
 	redef fun close do bytes = new Bytes.empty
 
-	redef fun read_all_bytes
```

Comment:
Why remove `read_all_bytes`?

Reply:
The super implementation should do the job alright in the first place, no need to redefine it here I'd think.
But come to think of it, in this particular case, it can produce a significant speedup, I'll leave the previous implementation available for this.